girder/api/rest.py

```diff
@@ -18,6 +18,7 @@
 ###############################################################################
 
 import cherrypy
+import cherrypy.lib
```

Comment:
Since we already have `cherrypy` top level module imported, do we need to explicitly import `cherrypy.lib`?